tempesta_fw/http2/hindex.h

```diff
@@ -0,0 +1,76 @@
+/**
+ *		Tempesta FW
+ *
+ * Indexing of the HPACK static and dynamic tables
+ * for headers compression.
+ *
+ * Copyright (C) 2017 Tempesta Technologies, Inc.
+ *
+ * This program is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License as published by
+ * the Free Software Foundation; either version 2 of the License,
+ * or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE.
+ * See the GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License along with
+ * this program; if not, write to the Free Software Foundation, Inc., 59
+ * Temple Place - Suite 330, Boston, MA 02111-1307, USA.
+ */
+
```

Comment:
Actually, it's tricky to build Linux kernel module code in a subdirectory within code in a parent directory. Thus, it's assumed that http2 content will be moved to tempesta_fw/ to be built as part of tempesta_fw.ko module. So please don't introduce too many small source code files and preferably name them as they will be in tempesta_fw/.